ReleaseTooling/CarthageJSON/FirebaseMLModelDownloaderBinary.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Hmm we currently don't build this in the zip, do we? I thought we had excluded it

Reply:
Doesn't look like it. It's part of 8.0.0. Perhaps we only excluded for Carthage before but now that Carthage matches zip, we shouldn't?

Reply:
Discussed offline, SGTM on including it.